tests/resources/test_access_requests.py

```diff
@@ -269,15 +273,19 @@ def test_access_grant_for_system_role(
     assert response.status_code == 403
 
     # Grant access to *any authenticated user*
-    service.access.create_grant(
+    service.access.create_grants(
```

Comment:
If this method existed in v11, then we introduce a breaking change. Can you check please? If so, then make sure we document this part for the future v12 release!